components/sections/Projects/ProjectTitle.jsx

```diff
@@ -0,0 +1,27 @@
+import Image from "next/image";
+
+const ProjectTitle = ({ name, openProjectPage, width, active, index }) => {
+
+  return (
+    <div
+      className="cursor-pointer relative group flex items-start"
+      onClick={width > 1024 ? openProjectPage: () =>{}}
+    >
+      <div className="text-13 leading-16 sm:text-21.5 sm:leading-18">{name}</div>
+      <div className={`transition-opacity duration-500 ${(active === index ? "opacity-100": "opacity-0")}`}>
```

Comment:
You don't need parenthesis on this line 
```suggestion
      <div className={`transition-opacity duration-500 ${active === index ? "opacity-100" : "opacity-0"}`}>
```